### DISCUSSION TITLE
Provide current binaries for Linux

### DISCUSSION THREAD

**Discussion by StarpTech** [containers/podman#15333] [Ideas]
As titled.

No AMD linux version is attached to the releases. In Ubuntu `podman` APT version is 1.23.1. Currently, there is no way to install the latest version of podman on Linux.

**Comment by afbjorklund** [comment]
This has been discussed a lot in other issues, but ubuntu decides what version of podman they want to ship in their distribution.

Currently, that is 3.x: https://packages.ubuntu.com/podman - there are some experimental packages in debian and in some PPA

**Reply by StarpTech** [reply]
Why can't we release the latest version through a custom PPA or via Snap?

**Comment by afbjorklund** [comment]
* https://github.com/containers/podman/issues/14302

**Comment by StarpTech** [comment]
What a nightmare :smile: Ever thought about a bash script that can install Podman on the most Linux machine?

**Reply by rhatdan** [reply]
Podman is a whole series of packages,  podman, fuse-overlayfs, crun, runc, netavark, aardvark, conmon, containers-common ...

If community wants to step up an support packaging for all the major distros, then they are welcome to do it.  Volunteers do it for Fedora and Debian right now, and I think perhaps a couple of others.

Sadly no one has stepped forward to do this for Ubuntu.

**Reply by StarpTech** [reply]
Yes, this is sad. That makes the adoption very hard. In comparison, installing docker is very easy. https://docs.docker.com/engine/install/

**Comment by lsm5** [comment]
@StarpTech just making sure you've seen https://github.com/containers/podman/issues/14302#issuecomment-1192617223 . I'm also currently checking with github actions people to use that repo: https://github.com/actions/runner-images/issues/5578

**Reply by StarpTech** [reply]
Thank you!
